### PR TITLE
Polyhedron demo: Fix Selection_io_plugin

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
This adds a dependency to selection_plugin to the selection_io_plugin, as the loading of a selection_item is actually performed by the selection_plugin.

### CODE REVIEW

**Comment by maxGimeno** [comment]
Maintenance is tested but shouldn't because of #3113

File: Polyhedron/demo/Polyhedron/Plugins/Point_set/Features_detection_plugin.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include <QAction>
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Plugins/Point_set/Features_detection_plugin.cpp]: Actually, that new error in ArchLinux is not related to this PR. ArchLinux has upgraded Qt from 5.10 to 5.11.

There are other compilation errors, in the demos of arrangements and linear
cell complex:
```
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:28:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:32:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:34:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:36:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:38:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:40:20: error: invalid use of incomplete type 'class QButtonGroup'
Arrangement_on_surface_2_Demo/NewTabDialog.cpp:48:27: error: invalid use of incomplete type 'class QButtonGroup'
Linear_cell_complex_Demo/MainWindow.cpp:55:31: error: invalid use of incomplete type 'class QHeaderView'
Linear_cell_complex_Demo/MainWindow.cpp:70:33: error: invalid use of incomplete type 'class QHeaderView'
Linear_cell_complex_Demo/MainWindow.cpp:70:69: error: incomplete type 'QHeaderView' used in nested name specifier
```

**Comment by maxGimeno** [comment]
I know the include errors are linked to Qt 5.11, but as trivial as they are to fix, it would be more effort to create a new PR for them than to put it in here, a PR that already concerns the 3D demos, so I find it more convenient like this.

**Comment by maxGimeno** [comment]
I did in a force push.

**Comment by lrineau** [comment]
As a general rule, now that Github is smarter about computing the history of PR,  I recommend to avoid forced push, unless that is really necessary. Because that breaks several things, including Github notifications.

**Comment by lrineau** [comment]
Successfully tested in [CGAL-4.13-Ic-30][*], and then merged.

  [*]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-30.shtml

**Pull request merged**